app/views/categories/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="w-full">
+  <% if notice.present? %>
+    <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
+  <% end %>
+
+  <div class="flex justify-between items-center">
+    <h1 class="font-bold text-4xl">Transactions</h1>
+    <%= link_to 'Back home', groups_path, class: "rounded-lg py-3 px-5 bg-blue-600 text-white block font-medium" %>
+    <%= link_to 'New category', new_category_path, class: "rounded-lg py-3 px-5 bg-blue-600 text-white block font-medium" %>
```

Comment:
[REQUIRED] Kindly move your `add new category` button to the bottom

Reply:
I think adding the button to the bottom would be quite inconveniencing especially if the user has many categories. It should be seen and readily available at the top